ci-operator/config/openshift-kni/eco-ci-cd/openshift-kni-eco-ci-cd-main__cnf-core-4.16.yaml

```diff
@@ -0,0 +1,85 @@
+base_images:
+  base:
+    name: "4.14"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    name: ci
+    namespace: openshift
+    tag: golang-1.20
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+  - as: eco-ci-cd
```

Comment:
Please look at my PR and fix the indentation here, the whole `- as ` block should be under the tests.

Reply:
not sure what do you mean, but `make update` did some changes )